cluster/swarm/utils.go

```diff
@@ -33,3 +34,13 @@ func convertMapToKVStrings(values map[string]*string) []string {
 	}
 	return result
 }
+
+var imageEngineOSErrorPattern = regexp.MustCompile(`cannot load (.+) image on (.+)`)
```

Comment:
@simonferquel @wsong is this pattern too general for image loading? I'm not aware of load errors so we should be sure about this.